### PR TITLE
Update re-using.md

### PR DESCRIPTION
**Pull request by korbinib**
toollist should include publication search engines, data deposition repositories, value added data DBs

### CODE REVIEW

File: pages/rdm_cycle/re-using.md [hunk 1]
```diff
@@ -1,8 +1,48 @@
 ---
 title: Re-using
 keywords:
+contributors:
```
Comment (floradanna) [inline pages/rdm_cycle/re-using.md:4]: contributors need to be added before merging

File: pages/rdm_cycle/re-using.md [hunk 2]
```diff
@@ -1,8 +1,48 @@
...
 ---
 
-## Subtitle 1
+## What is Data Re-use?
 
-Some text that you can fill in
\ No newline at end of file
+Data re-use describes the process of using available data for own, new research.
```
Comment (floradanna) [inline pages/rdm_cycle/re-using.md:9]:
```suggestion
Data reuse describes the process of using available data for own, new research.
```

File: pages/rdm_cycle/re-using.md [hunk 3]
```diff
@@ -1,8 +1,48 @@
...
+
+## Why Data Re-use is important?
+
+By re-using existing data you can:
```
Comment (floradanna) [inline pages/rdm_cycle/re-using.md:13]:
```suggestion
By reusing existing data you can:
```

File: pages/rdm_cycle/re-using.md [hunk 4]
```diff
@@ -1,8 +1,48 @@
...
+
+* obtain reference data for your research
+* avoid doing new, unecessary experiments
+* make research more robust by aggregating results obtained from different methods or samples
+* gain novel insights by connecting and meta-analysing data-sets
```
Comment (floradanna) [inline pages/rdm_cycle/re-using.md:18]:
```suggestion
* gain novel insights by connecting and meta-analysing datasets
```

**Review by korbinib — COMMENTED** [review]
added contributors

File: pages/rdm_cycle/re-using.md [hunk 5]
```diff
@@ -1,8 +1,48 @@
...
+
+## What should be considered for Data Re-use?
+
+One major challenge is to find data for re-use. Another point is that you should assess the quality of the deposited data before it is re-used.
+One starting point can be to use value added databases with curatated content. Other possibities include searching data deposition repositories for suitable datasets based on their annotation or obtaining data directly from the author. You should keep in mind whether the repositories provide sufficient metadata to enable data re-use. Some types of data you are searching for can be straightforward to re-use (e.g. genome data) or require extensive metadata to interpret and reuse (e.g. gene expression experiment data)
```
Comment (smza) [inline pages/rdm_cycle/re-using.md:23]: correct typo: curated

File: pages/rdm_cycle/re-using.md [hunk 6]
```diff
@@ -1,8 +1,48 @@
...
+The re-use of data should be aligned with the intellectual property right e.g. by making sure it is available under a suiting license.
```
Comment (smza) [inline pages/rdm_cycle/re-using.md:24]: Correct suiting license to "suitable license"

**Review by smza — COMMENTED** [review]
Minor typographical and grammatical  corrections suggested.

File: pages/rdm_cycle/re-using.md (hunk 6 above)
Comment (korbinib) [reply pages/rdm_cycle/re-using.md:24]: Thanks!

File: pages/rdm_cycle/re-using.md (hunk 5 above)
Comment (korbinib) [reply pages/rdm_cycle/re-using.md:23]: Thanks!

**Pull request merged**